docs/guide/in-depth/advanced-chunks.md

```diff
@@ -65,3 +297,57 @@ When we run `node ./main.js`, the traversal order of the modules would be `main.
 `second.js` tries to call `__esm` function before it gets initialized. This will lead to a runtime error which is trying to call `undefined` as a function.
 
 With forcefully generated `runtime.js`, the bundler ensures any chunk that depends on runtime code would first load `runtime.js` before executing itself. This guarantees that the runtime code is always executed before any other chunks, preventing circular import issues.
+
+### Why the dependencies of the captured module get captured too?
+
+When a module is captured by a group, rolldown will try to capture its dependencies recursively. This is because rolldown is only allowed to mangling exports of non-entry chunks by default.
```

Comment:
```suggestion
When a module is captured by a group, Rolldown will try to capture its dependencies recursively. This is because Rolldown is only allowed to mangle the exports of non-entry chunks by default.
```